src/Tests/dotnet-run.Tests/GivenDotnetRunRunsCsProj.cs

```diff
@@ -379,12 +380,18 @@ public void ItGivesAnErrorWhenAttemptingToUseALaunchProfileThatDoesNotExistWhenT
 
             var testProjectDirectory = testInstance.Path;
 
-            new DotnetCommand(Log, "run")
+            var runResult = new DotnetCommand(Log, "run")
                 .WithWorkingDirectory(testProjectDirectory)
-                .Execute("--launch-profile", "test")
-                .Should().Pass()
-                         .And.HaveStdOutContaining("Hello World!")
-                         .And.HaveStdErrContaining(LocalizableStrings.RunCommandExceptionCouldNotLocateALaunchSettingsFile);
+                .Execute("--launch-profile", "test");
+
+            string[] expectedErrorWords = LocalizableStrings.RunCommandExceptionCouldNotLocateALaunchSettingsFile.Replace("\'{0}\'", "").Split(" ");
```

Comment:
Would it be simpler just to format the message with the expected parameters and verify that the output contains that, instead of checking that it has all the words (but not verifying order or anything)?

Reply:
This is a good option, though it would require extracting `testRoot`, specifying the TFM, and hardcoding parts of the launchSettings path into the test. I prefer this option for this test since that information isn't relevant to the test, makes the test more flexible to changes in the asset, and can be done in less lines of code